spopt/region/util.py

```diff
@@ -747,6 +743,14 @@ def _randomly_divide_connected_graph(adj, n_regions):
             f"equal to the number of nodes which is {n_areas}."
         )
     mst = csg.minimum_spanning_tree(adj)
+    mst_copy = mst.copy()
```

Comment:
I think you can use a classic trick here: 

`symmetric_mst = (mst + mst.T) > 0`

Reply:
@ljwolf How does this look?

Reply:
This is so smooth!

Reply:
@gegen07 I know, right? Always some tricks to learn that I have never seen!